src/pages/ExcelOpplastning.tsx

```diff
@@ -249,11 +249,12 @@ const ExcelOpplastning = () => {
             </Lenke>
           </AlertStripeAdvarsel>
           <Ingress className="container">
-            Vanligvis skal arbeidsgiveren betale sykepenger de første
-            16 kalenderdagene (arbeidsgiverperioden) av et sykefravær.
-            I forbindelse med korona-pandemien kan arbeidsgiveren søke om refusjon
-            fra og med fjerde dag i arbeidsgiverperioden.
-            Dette gjelder hvis den ansatte enten er smittet, mistenkt smittet eller i pålagt karantene.
+            Når sykefraværet handler om korona, dekker NAV sykepenger fra dag 4 i de 16 dagene arbeidsgiveren vanligvis
```

Comment:
container klassen hører til grid/layout. Foreslår endre det til:
&lt;Container>
&lt;Ingress>

Reply:
Den er annerledes enn begge de to andre igjen 😂